opm/upscaling/RelPermUtils.cpp

```diff
@@ -888,7 +888,7 @@ void RelPermUpscaleHelper::upscaleCapillaryPressure()
         to_double(options["saturationThreshold"]);
 
     auto largestSaturationInterval = Swor - Swir;
-    auto Ptestvalue                = Pcmax;
+    decltype(Pcmax) Ptestvalue;
```

Comment:
This was a bit weird, can you explain why this is needed?

Reply:
Unused assignment. Apparently type deduction was wanted and i cant declare a bare auto.